pkg/certificates/certificate_manager.go

```diff
@@ -50,19 +50,27 @@ type CertManager struct {
 	APIServiceName        string
 	Logger                logr.Logger
 	Ready                 chan struct{}
+	EnableWebhookPatching bool
 }
 
 // AddCertificateRotation registers all needed services to generate the certificates and patches needed resources with the caBundle
 func (cm CertManager) AddCertificateRotation(ctx context.Context, mgr manager.Manager) error {
-	var rotatorHooks = []rotator.WebhookInfo{
-		{
-			Name: cm.ValidatingWebhookName,
-			Type: rotator.Validating,
-		},
-		{
-			Name: cm.APIServiceName,
-			Type: rotator.APIService,
-		},
+	var rotatorHooks []rotator.WebhookInfo
+
+	rotatorHooks = append(rotatorHooks, rotator.WebhookInfo{
+		Name: cm.APIServiceName,
+		Type: rotator.APIService,
+	})
```

Comment:
nit: saving a few CPU cycles :)
```suggestion
	rotatorHooks := []rotator.WebhookInfo{
        {
		    Name: cm.APIServiceName,
		    Type: rotator.APIService,
	    },
	}
```